src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -4552,9 +4552,17 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
         // set supplemental heating coil operation temperature limits
         if (thisVrfTU.SuppHeatingCoilPresent) {
             // Set maximum supply air temperature for supplemental heating coil
-            thisVrfTU.MaxSATFromSuppHeatCoil = rNumericArgs(11);
+            if (NumNums < 11) {
+                thisVrfTU.MaxSATFromSuppHeatCoil = DataSizing::AutoSize;
+            } else {
+                thisVrfTU.MaxSATFromSuppHeatCoil = rNumericArgs(11);
+            }
             // set maximum outdoor dry-bulb temperature for supplemental heating coil operation
-            thisVrfTU.MaxOATSuppHeatingCoil = rNumericArgs(12);
+            if (NumNums < 12) {
+                thisVrfTU.MaxOATSuppHeatingCoil = 21.0;
+            } else {
+                thisVrfTU.MaxOATSuppHeatingCoil = rNumericArgs(12);
+            }
```

Comment:
This code was changed in #9768 and somehow was reverted. While here also added the default value for MaxSATFromSuppHeatCoil.

Reply:
Or maybe this is what inspired the GetObjectItem/default testing...?

Reply:
This was prior to. Probably don't need the "NumNums <" anymore.